### ISSUE TITLE
ocpindex.vim set to wrong location in .vimrc

### ISSUE THREAD

**Issue by struktured** [OCamlPro/opam-user-setup#18]
When I start vim, I get this error after running `opam install user-setup`:

```
Error detected while processing function OpamConfOcpIndex:
line    2:
E484: Can't open file /home/carm/.opam/4.02.3/share/vim/syntax/ocpindex.vim
```

This is my work around at the moment:
`ln -sf `opam config var share`/ocp-index/vim/autoload/ocpindex.vim `opam config var share`/vim/syntax/ocpindex.vim`

**Comment by AltGr** [comment]
there has been a change of location, what combination of versions are you using ?
latest ocp-index requires trunk opam-user-setup at the moment (just waiting for some more tests before pushing a new release! -- so your feedback is very welcome)

**Comment by rjbou** [comment]
Fixed in version 0.7. Thanks for reporting!
